src/cljs/hakukohderyhmapalvelu/views/hakukohderyhma_settings_view.cljs

```diff
@@ -56,8 +56,31 @@
    (when (get-in selected-ryhma [:settings :rajaava])
      (max-hakukohteet selected-ryhma))])
 
+(defn- jyemp-checkbox
+  [selected-ryhma]
+  [:div (stylefy/use-style {:display        "flex"
+                            :flex-direction "row"
+                            :margin-top     "1rem"})
+    (checkbox/checkbox-slider {:checked?        (get-in selected-ryhma [:settings :jos-ylioppilastutkinto-ei-muita-pohjakoulutusliitepyyntoja])
+                               :cypressid       "jyemp-checkbox"
+                               :id              "jyemp-checkbox"
+                               :aria-labelledby @(subscribe [:translation :hakukohderyhma/jyemp])
+                               :on-change       (fn []
+                                                  (let [settings (:settings selected-ryhma)
+                                                        jyemp (not (get-in selected-ryhma [:settings :jos-ylioppilastutkinto-ei-muita-pohjakoulutusliitepyyntoja]))
```

Comment:
`jyemp (not (:jos-ylioppilastutkinto-ei-muita-pohjakoulutusliitepyyntoja settings))`